src/code/z_scene_table.c

```diff
@@ -1633,7 +1652,7 @@ void (*sSceneDrawConfigs[SDC_MAX])(PlayState*) = {
 };
 
 void Scene_Draw(PlayState* play) {
-    if (R_HREG_MODE == HREG_MODE_SCENE_CONFIG) {
+    if ((R_HREG_MODE == HREG_MODE_SCENE_CONFIG) && OOT_DEBUG) {
```

Comment:
fairly sure OOT_DEBUG first works, at least it did when i started this file the other day

Reply:
Having it first matches retail, but then breaks debug on my end. Did you check debug as well when you did it?
![image](https://github.com/zeldaret/oot/assets/47598039/56d57695-1ab5-47dc-a507-ccdfe747402c)

Reply:
probably not, no. that sucks